mm2src/mm2_main/src/lp_ordermatch.rs

```diff
@@ -5221,6 +5221,30 @@ pub enum CancelBy {
     Coin { ticker: String },
 }
 
+pub async fn get_matching_orders(ctx: &MmArc, ticker: &str) -> Vec<Uuid> {
+    let mut matching_orders = vec![];
+    if let Ok(ordermatch_ctx) = OrdermatchContext::from_ctx(ctx) {
+        let maker_orders = ordermatch_ctx.maker_orders_ctx.lock().orders.clone();
+        let taker_orders = ordermatch_ctx.my_taker_orders.lock().await;
+
+        for (uuid, order) in maker_orders.iter() {
+            let uuid = *uuid;
+            let order = order.lock().await.clone();
+            if order.base == ticker || order.rel == ticker && !order.is_cancellable() {
```

Comment:
Should be `if (order.base == ticker || order.rel == ticker) && !order.is_cancellable()` `&&` has precedence over `||`

Reply:
Ok 👍🏽